src/Controller/ProjectViewController.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the Kimai time-tracking app.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Controller;
+
+use App\Entity\Project;
+use App\Event\ProjectMetaDisplayEvent;
+use App\Repository\ProjectRepository;
+use App\Repository\Query\ProjectQuery;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route(path="/project_view")
+ */
+final class ProjectViewController extends AbstractController
```

Comment:
I'm not sure which permissions to require here. Do I need to define a new or use an existing one?

Reply:
I'd say project_budget or project_details can be reused